packages/genesys-spark-components/src/components/beta/gux-avatar-group/example.html

```diff
@@ -0,0 +1,71 @@
+<h2>Avatar Groups</h2>
+
+<h3>Avatars</h3>
+<gux-avatar-group-beta>
+  <gux-avatar-beta name="Alan Butler"
+    ><img slot="image" src="https://i.pravatar.cc/300" alt="Alan Butler"
+  /></gux-avatar-beta>
+  <gux-avatar-beta name="Conor Darcy"></gux-avatar-beta>
+  <gux-avatar-beta name="Elliot Fitzgerald"></gux-avatar-beta>
+  <gux-avatar-beta name="Greg Hayes">
+    <img slot="image" src="https://i.pravatar.cc/300" alt="Greg Hayes"
+  /></gux-avatar-beta>
+  <gux-avatar-beta name="Iseult Jones"></gux-avatar-beta>
+  <gux-avatar-beta name="John King"></gux-avatar-beta>
+</gux-avatar-group-beta>
```

Comment:
There could be a use case for focus at a group level similar to this [SAP Avatar Groups](https://experience.sap.com/fiori-design-web/avatar-group-web-component/#anatomy1) but I don't think it needs to go in with this.